src/main/scala/com/github/mlangc/slf4zio/api/package.scala

```diff
@@ -71,6 +70,56 @@ package object api {
         logger.error(msg, th)
     }
 
+    def traceIO(marker: Marker, msg: => String): UIO[Unit] = UIO {
+      if (logger.isTraceEnabled(marker))
+        logger.trace(marker, msg)
+    }
+
+    def debugIO(marker: Marker, msg: => String): UIO[Unit] = UIO {
+      if (logger.isDebugEnabled(marker))
+        logger.debug(marker, msg)
+    }
+
+    def infoIO(marker: Marker, msg: => String): UIO[Unit] = UIO {
+      if (logger.isInfoEnabled(marker))
+        logger.info(marker, msg)
+    }
+
+    def warnIO(marker: Marker, msg: => String): UIO[Unit] = UIO {
+      if (logger.isWarnEnabled(marker))
+        logger.warn(marker, msg)
+    }
+
+    def errorIO(marker: Marker, msg: => String): UIO[Unit] = UIO {
+      if (logger.isErrorEnabled(marker))
+        logger.error(marker, msg)
+    }
+
+    def traceIO(marker: Marker, msg: => String, th: Throwable): UIO[Unit] = UIO {
+      if (logger.isTraceEnabled(marker))
+        logger.trace(marker, msg, th)
+    }
+
+    def debugIO(marker: Marker, msg: => String, th: Throwable): UIO[Unit] = UIO {
+      if (logger.isDebugEnabled)
+        logger.debug(marker, msg, th)
+    }
+
+    def infoIO(marker: Marker, msg: => String, th: Throwable): UIO[Unit] = UIO {
+      if (logger.isInfoEnabled)
+        logger.debug(marker, msg, th)
```

Comment:
I guess you meant `info` here.

Reply:
Thanks! The same is for line 60. I've just copy-pasted. Will fix.